ospeak/cli.py

```diff
@@ -7,14 +7,15 @@
 from pydub.playback import play
 
 VOICES = ["alloy", "echo", "fable", "onyx", "nova", "shimmer"]
+MODELS = ["tts-1", "tts-1-hd"]
```

Comment:
The [current list for my account](https://github.com/tomviner/scrape-ai/blob/main/openai/api/models.txt) contains:
```
canary-tts
tts-1
tts-1-1106
tts-1-hd
tts-1-hd-1106
```

This is subject to arbitrary change, so I wonder if it might be best to avoid a hardcoded list. Instead, catch the missing model error:
```
openai.NotFoundError: Error code: 404 - {'error': {'message': 'The model `dummy` does not exist', 'type': 'invalid_request_error', 'param': None, 'code': 'model_not_found'}}
```